taxi/models.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+    class Meta:
+        verbose_name_plural = "All manufacturers"
+
+    def __str__(self):
+        return f"{self.name} {self.country}"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(
+        max_length=100,
+        unique=True,
+    )
+
+    class Meta:
+        verbose_name_plural = "All drivers"
+
+    def __str__(self):
+        return f"{self.username} {self.license_number}"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=65)
+    manufactured = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.SET_NULL,

```

Comment:
why `set_null`?

Reply:
I think if car manufacturer will be deleted, car should not be deleted

Reply:
I think that cars cannot be without a manufacturer. So if we delete a manufacturer cars should be also deleted